clone_nonbillables_and_process.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -xe
+
+# Add deploy key to ssh config
+mkdir -p ~/.ssh
+touch ~/.ssh/config
+touch ~/.ssh/id_nonbillable
+if ! grep -q "Host github-nonbillable" ~/.ssh/id_nonbillable; then
+    echo "
+    Host github-nonbillable
+        HostName github.com
+        IdentityFile ~/.ssh/id_nonbillable
+    " > ~/.ssh/config
+    echo "$GH_NONBILLABLE_DEPLOYKEY" > ~/.ssh/id_nonbillable
+fi
+chmod 600 ~/.ssh/id_nonbillable
+
+git clone git@github-nonbillable:CCI-MOC/non-billable-projects.git ~/non-billable-projects
+cp ~/non-billable-projects/*.txt .
```

Comment:
You shouldn't do this. Treat the repository as the single source of truth and pass-in the path to the files inside the folder when calling `process_csv_report`.

Reply:
I have removed the `cp`. As for passing the files into `process_report.py`, I'll be handling that part in #21.